act-framework.bs

```diff
@@ -63,7 +63,7 @@ ACT Test Procedure {#test-proc}
 Selector {#test-proc-selector}
 --------
 
-A selector is a boolean predicate that takes an element or component in the set represented in the input data and unambiguously tests whether the element or component matches the selector or not. In an ACT Rule, a selector is applied to the entire input data to filter it into a set of elements or components to be evaluated with the test procedure.
+A selector is a pattern used to select element(s) in an HTML document or node(s) in the DOM tree. In an ACT Rule, a selector is applied to the entire input data to filter it into a set of elements or nodes to be evaluated against the test procedure.
```

Comment:
We shouldn't limit ourselves to HTML document or DOM Tree. Instead. I'd suggest:

`A selector is a procedure for locating elements in a web page that will be tested by the ACT Rule. For example, finding all img nodes in the DOM tree, or finding tags that are incorrectly closed in a HTML document. A selector is applied to the entire input data, gather from it, a set of elements or nodes to be evaluated against the test procedure.`